### PR TITLE
Feat/storage

### PR DESCRIPTION
**Pull request by Gabester0**
Will need to add something to handle the conversion of very large numbers as JavaScript cannot reliably handle numbers larger than 2 to the 53rd power - 1

### CODE REVIEW

File: server-api/data/storage.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+units['gib'] = {unit:'gib', base: 'b', name: 'gibibyte', type: 'storage', factor: Math.pow(2, 30), dataSource: "https://physics.nist.gov/cuu/Units/binary.html" };
+units['tib'] = {unit:'tib', base: 'b', name: 'tebibyte', type: 'storage', factor: Math.pow(2, 40), dataSource: "https://physics.nist.gov/cuu/Units/binary.html" };
+units['pib'] = {unit:'pib', base: 'b', name: 'pebibyte', type: 'storage', factor: Math.pow(2, 50), dataSource: "https://physics.nist.gov/cuu/Units/binary.html" };
+units['eib'] = {unit:'eib', base: 'b', name: 'exibyte', type: 'storage', factor: Math.pow(2, 60), dataSource: "https://physics.nist.gov/cuu/Units/binary.html" };
+units['zib'] = {unit:'zib', base: 'b', name: 'zebibyte', type: 'storage', factor: Math.pow(2, 70), dataSource: "https://www.geeksforgeeks.org/understanding-file-sizes-bytes-kb-mb-gb-tb-pb-eb-zb-yb/" };
+units['yib'] = {unit:'yib', base: 'b', name: 'yobibyte', type: 'storage', factor: Math.pow(2, 80), dataSource: "https://www.geeksforgeeks.org/understanding-file-sizes-bytes-kb-mb-gb-tb-pb-eb-zb-yb/" };
+
+//? Will need Math library for this
```
Comment (DonaldMurillo) [inline server-api/data/storage.ts:25]: you can write 2**10 no?

File: server-api/data/storage.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import { Units } from '../models/converter.model';
+
+const units: Units = {};
+
+units['b'] = {unit:'b', base: 'b', name: 'byte', type: 'storage', factor: 1, dataSource: "https://physics.nist.gov/cuu/Units/binary.html" };
```
Comment (DonaldMurillo) [inline server-api/data/storage.ts:6]: you are missing the a "bit" =)

File: server-api/data/storage.ts (hunk 1 above)
Comment (Gabester0) [reply server-api/data/storage.ts:25]: Lol, yeah, I didn't realize that. I will update

File: server-api/data/storage.ts (hunk 2 above)
Comment (Gabester0) [reply server-api/data/storage.ts:6]: Ok, I added this in.  Let me know if there is anything else

**Review by DonaldMurillo — APPROVED** [review]

**Pull request merged**